Client/Frontend/Browser/HomePanel/favorites/FavoritesDataSource.swift

```diff
@@ -76,7 +76,8 @@ class FavoritesDataSource: NSObject, UICollectionViewDataSource {
             return UICollectionViewCell()
         }
         
-        let ftd = FavoritesTileDecorator(url: url, cell: cell, indexPath: indexPath, color: fav.color != nil ? UIColor(colorString: fav.color!) : nil)
+        // BRAVE TODO: Update Favorites logic, color argument is outdated here. Setting nil for now.b
```

Comment:
Is there at ticket to track this?
Also, remove extraneous "b"

Reply:
This was fixed in https://github.com/brave/brave-ios/pull/383, I only need to merge latest development changes to this rbanch